L1Trigger/L1TMuonEndCapPhase2/BuildFile.xml

```diff
@@ -0,0 +1,13 @@
+<export>
+    <lib name="1"/>
+</export>
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="DataFormats/L1TMuon"/>
+<use name="DataFormats/L1TMuonPhase2"/>
+<use name="DataFormats/RPCRecHit"/>
+<use name="L1Trigger/L1TMuon"/>
+<use name="PhysicsTools/TensorFlow"/>
+<use name="hls" />
+<flags LDFLAGS="-lXMLIO"/>
```

Comment:
What is the purpose of this line? We generally want to deal with library dependencies via `<use ... />` elements than altering `LDFLAGS`.